app/views/books/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2> New Book </h2>
+  <%= form_for @book, action: 'create', method: :post do |f| %>
+<fieldset>
```

Comment:
Watch your indenting here. ```<fieldset>``` should be indented inside the ```<%= form_for %>``` tag. It may seem small, but consistent formatting goes a long way towards being able to read code quickly.